azavea/raster-foundry
# Introduce new project export pane

## Overview

This front-end work allows for export of projects to reference a tool-run. This does not include the listing of a user's exports.gcd

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo

With only implemented export settings shown:
<img width="568" alt="screen shot 2017-06-15 at 10 36 17 am" src="https://user-images.githubusercontent.com/2442245/27186619-e9c42ea0-51b6-11e7-9bea-92ad4b00f22c.png">

With all export settings shown:
<img width="507" alt="screen shot 2017-06-15 at 10 37 42 am" src="https://user-images.githubusercontent.com/2442245/27186625-ed59db78-51b6-11e7-9b74-89c889099636.png">

With export option 'previews':
<img width="877" alt="screen shot 2017-06-15 at 10 37 52 am" src="https://user-images.githubusercontent.com/2442245/27186658-06b12de2-51b7-11e7-8224-eff1679eba3c.png">

Without export option 'previews':
<img width="851" alt="screen shot 2017-06-15 at 10 38 48 am" src="https://user-images.githubusercontent.com/2442245/27186664-08693ec2-51b7-11e7-8242-324596431f19.png">


### Notes

There are a lot of things to think about here, but the one that gave me the most pause was how we should approach defining which tools are applicable to a project export. Some tools make sense for a single project while some do not.

## Testing Instructions

Try some exports, but more specifically:
* try an export of NDVI and make sure that a tool-run was POSTed, then make sure that tool run id was indicated in the export POST
* try a regular export (color corrected or raw) and see that the export was POSTed successfully

Closes #1780 
Closes #1778 
Closes #1779
